engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details-info.controller.js

```diff
@@ -42,6 +43,22 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
             return deferred.promise;
         };
 
+        $scope.architectures = function () {
+            var deferred = $q.defer();
+            Architecture.queryUnpaged(function (architectures) {
+                var results = architectures.results;
+                results.map(function(i) {
+                    i.id = i.name;
+                });
+                results.push({
+                    id: 'noarch',
+                    name: 'noarch (Unrestricted)'
+                });
```

Comment:
Couple of issues with this:

1) when creating a new repo, i'd expect the default value of 'noarch (unrestricted)' to be selected, instead nothing is selected.
2) it looks like if i select 'noarch (unrestricted)' it actually saves it as 'noarch (unrestricted)' instead of just 'noarch'.  This causes things to not work in candlepin.